theme/DesignTheme/Views/NewsItem.scss

```diff
@@ -0,0 +1,22 @@
+.full-width {
+  position: relative;
+  right: 50%;
+  left: 50%;
+  width: 100vw !important;
+  max-width: initial !important;
+  margin-right: -50vw !important;
+  margin-left: -50vw !important;
+  object-fit: cover;
+  height: 480px;
+}
+
+.newsitem-view {
+  #text-body {
+    h2,
+    h3,
+    h4,
+    h5 {
+      font-family: 'Titillium Web', Geneva, Tahoma, sans-serif;
```

Comment:
Sono stupito che serva definire questa cosa, come mai?

Reply:
@pnicolli è il primo modo che mi è venuto in mente: la classe per vedere il Lora nei richtext la metto nel wrapper dei richtet. Se il redattore scrive dei titoli nel richtext, questi sono in Lora e non in Titillium...Per cui l'ho ridefinito... modi più furbi?

Reply:
Ah quindi se non metti questo, gli h* non vengono in Titillium? Non l'ho provato in effetti. In tal caso ok,

Reply:
@pnicolli già